lms/static/js/views/base_dashboard_view.js

```diff
@@ -0,0 +1,10 @@
+(function(define) {
```

Comment:
Perhaps some documentation at the top of this file explaining why one would use this base view?